### ISSUE TITLE
[Initiative] Post Updates to Social Media

### ISSUE THREAD

**Issue by D1M1TR10S** [ersilia-os/ersilia#282]

**Comment by sayantani11** [comment]
@D1M1TR10S Like do we want automation or manual?

**Comment by D1M1TR10S** [comment]
Hi @sayantani11 ! I'll whatever details I have in the top-level comment.

To answer your question: Yes! We want this process of posting to Social Media to be automated with GitHub Actions.

So every time a new submission is approved and  deployed to Ersilia's CLI, we want to trigger an event that posts to platforms like Twitter, LinkedIn, etc.

**Comment by GemmaTuron** [comment]
Hi @D1M1TR10S  and @megamanics thanks for taking the lead on this!
I 'd start with 
- Twitter
- Slack

What do you need from us at this point?

**Comment by D1M1TR10S** [comment]
Sure thing @GemmaTuron. We're going to kick this off once the new Model Submissions Flow is complete, since this would depend on that workflow.

For the kickoff, here's what I think we'll need:
- Ersilia's login info for the Twitter and Slack accounts
- Either grant admin access to @megamanics on Twitter and Slack so he can get the API tokens for each platform, and set them up as endpoints for the workflows. _Or if you prefer not to grant access, you can pair with Ski to share the API tokens + any other important info required for the integration. Granting access might make it easier for Ski to test the workflows._
- Align on copy for the automated posts. (What are they going to say?)
- Define the trigger event kicking off this workflow. (Will it be a new PR being merged? Or a deployment of a new model?)

Does that all sound right @megamanics?

**Comment by D1M1TR10S** [comment]
Once the Airtable dependency is removed, this workflow will be ready to continue working on.

We can go through the approval process when there's a new PR. Once the PR works on local, we can initiate a Slack message.

**Comment by GemmaTuron** [comment]
- [ ] Create a template of the post (what information is there) -- @miquelduranfrigola and @GemmaTuron 
- [ ] Create a workflow dispatch
- [ ] The action will be manually triggered

**Comment by D1M1TR10S** [comment]
Before posting to the public (like LinkedIn or Twitter) we'll need to have automated testing in Slack. These two platforms can be deprioritized for the current scope of work.

### Save for the next round
_Can be done manually in the meantime_
- **Possible:** can create content on Medium dynamically based on what each model does. Add a template structure to the input form on the template repo: "What does this model do? Who is it for? How does it work?"
- Then automatically post this blog article to Twitter and LinkedIn. (might be best to do less posts, but with higher quality)

**Comment by GemmaTuron** [comment]
For the issue creation in GitHub, we need to create a list of users that will be selected randomly to test a model once it is ready

**Comment by megamanics** [comment]
@GemmaTuron List of test users can be updated using the below url


https://github.com/organizations/ersilia-os/settings/variables/actions
![Image](https://user-images.githubusercontent.com/10250297/219907783-9878e228-611e-48c8-b19b-926008a1dc55.png)
